blog/2024-01-23/index.md

```diff
@@ -0,0 +1,58 @@
+---
+slug: over-one-million-active-users-and-growing
+title: Over One Million Active Users—and Growing
+authors: [cassidyjames]
+tags: [flathub, milestone, stats]
+---
+
+Earlier this month we shared [new app metadata guidelines](../2024-01-08/index.md) in response to the growth and maturity of Flathub. Today we're proud to share about that growth in a bit more detail including a huge milestone, how we calculate stats, and what we believe is driving that growth.
+
+<!-- truncate -->
+
+## Milestones
+
+To date, Flathub has served a total of **1.6 billion downloads** of **over 2,400 apps** and their updates.
+
+Since introducing [verified apps](/docs/for-users/verification) last year, **over 800 apps have been verified** by their original authors—more than one third of all apps, with that number constantly increasing. 
```

Comment:
Hrm, that dash thing might be something you do, which I don't get :D

Reply:
At least put some spaces next to it?

Reply:
At least in US English, em-dahses don't have spaces. Using them at all is stylistic, I suppose—and yeah, I probably overuse them… 😉 

https://www.thesaurus.com/e/grammar/em-dash/